ansible_collections/juniper/device/plugins/connection/pyez.py

```diff
@@ -732,7 +751,8 @@ def load_configuration(self, config, load_args):
                                (str(ex)))
 
     def commit_configuration(self, ignore_warning=None, comment=None,
-                             confirmed=None):
+                             confirmed=None, timeout=30, full=False,
```

Comment:
Is the default timeout of 30sec common across all operations? Can there be a constant defined for it and used?

Reply:
RPC execution time can vary considerably depending on the RPC and the device. By default, NETCONF RPCs time out after 30 seconds.

https://www.juniper.net/documentation/us/en/software/junos-pyez/junos-pyez-developer/topics/task/junos-pyez-rpcs-executing.html